roaster-tm4j/roaster-tm4j-dto/pom.xml

```diff
@@ -0,0 +1,91 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>roaster-tm4j</artifactId>
+        <groupId>hu.icellmobilsoft.roaster</groupId>
+        <version>0.2.0-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>roaster-tm4j-dto</artifactId>
+
+    <dependencies>
+        <dependency>
+            <groupId>hu.icellmobilsoft.coffee</groupId>
+            <artifactId>coffee-dto-base</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.eclipse.microprofile.openapi</groupId>
+            <artifactId>microprofile-openapi-api</artifactId>
+            <scope>provided</scope>
```

Comment:
Ennel a projektnel szerintem a compile kellene, mivel keretrendszerrol van szo, nem nyujta az alkalmazas szerver ennek az implementaciojat, de beszelhetunk rola...

Reply:
A dto modulba csak a fordításhoz kell, hogy a `@Schema` annotációt ismerje. Runtime ott van ez a lib, ahol ezt a modult felhasználják. Működik így is, de biztos ami biztos, akkor legyen compile scope-ú.